cdc/processor/pipeline/sink.go

```diff
@@ -71,7 +71,8 @@ type sinkNode struct {
 	status  TableStatus
 	tableID model.TableID
 
-	resolvedTs   model.Ts
+	// atomic oprations for model.ResolvedTs
+	resolvedTs   atomic.Value
```

Comment:
Prefer it to be the clear type. https://rust.godbolt.org/z/5qKjjPx64 Also it seems to need more instructions. And there is an additional type assertion.